rust-connectors/sources/mqtt/src/error.rs

```diff
@@ -16,12 +17,16 @@ pub enum MqttConnectorError {
     MqttConnection(#[from] MqttConnectionError),
     #[error("Mqtt client error: `{0:#?}`.")]
     MqttClient(#[from] MqttClientError),
+    #[error("Mqtt OptionError: `{0:#?}`.")]
+    MqttOption(#[from] MqttOptionError),
     #[error("Fluvio error: `{0:?}`.")]
     Fluvio(#[from] FluvioError),
     #[error("Clap Error: `{0:#?}`.")]
     Clap(#[from] ClapError),
     #[error("Serde Json Error: `{0:#?}`.")]
     SerdeJson(#[from] SerdeJsonError),
+    #[error("URL Parse Error: `{0:#?}`.")]
```

Comment:
Do we need the debug representation here? I wonder if this could leak any secrets

Reply:
Ah good catch.